.github/CODEOWNERS

```
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj
+* @ameliav @dav3r @felddy @jasonodoom @jcantu248 @jsf9k @mcdonnnj @nickviola @schmelz21
```

Since you have a dev team for this and we are not involved in this code, you don't need us as code owners:
```suggestion
* @ameliav @jcantu248 @nickviola @schmelz21
```